content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,623 @@
+      "email": {
+        "to": "dev@sensuapp.io"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+In this instance, although we are using only one handler **email**, the handler is ran multiple times to send an email one to "support@sensuapp.io" and one to "dev@sensuapp.io":
+
+![Single Handler with Multiple Matching Contacts](/images/contact-routing/single-handler/multiple-matching-contacts.png)
+
+### Single Handler with Multiple Non-Matching Contacts
+
+For this configuration we have a single handler **email** and multiple contacts, **support** and **dev** configured. The "support" contact has an override configuration for **slack** and the **dev** contact has an override configuration for **pagerduty**, neither of which have an override configuration for **email**.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email"
+      ],
+      "contacts": [
+        "support",
+        "dev"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "slack": {
+        "channel": "#support"
+      }
+    },
+    "dev": {
+      "pagerduty": {
+        "service_key": "foobarkey"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+In this instance, since the contacts defined do not have an override configuration for **email** the default configuration for **email** will be used, in our example an email will be sent to "default@example.com"
+
+![Single Handler with Multiple Non-Matching Contacts](/images/contact-routing/single-handler/multiple-non-matching-contacts.png)
+
+### Single Handler with Some Matching Contacts
+
+In this configuration we have a single handler **email** and multiple contacts **support**, **dev** and **eng** configured. The **support** contact has an override configuration for **email**, the **dev** contact has an override configuration for **pagerduty** and the **eng** contact has an override configuration for **slack**.
+
+Since two contacts 
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email"
+      ],
+      "contacts": [
+        "support",
+        "dev"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.io"
+      }
+    },
+    "dev": {
+      "pagerduty": {
+        "service_key": "foobarkey"
+      }
+    },
+    "eng": {
+      "slack": {
+        "channel": "#engineering"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+For this instance two emails will be generated. The first email will be sent to **support** contact at "support@sensuapp.io". The second email will be sent to our default email at "default@exmaple.com".
+
+The reason for this is that since we have one or more contacts that do not have a configuration override for **email**, we use the default configuration.
+
+![Single Handler with Some Matching Contacts](/images/contact-routing/single-handler/some-matching-contacts.png)
```

Comment:
Image shows both emails going to support@sensuapp.io.

I am also generally curious about this configuration, which I can test on my own. I wasn't aware that having multiple contacts defined and if no match was found for one of them for that handler, it would send an email anyways to the email address defined in the email handler as well as the matching contact.

Reply:
correct, if you have 1 or more contacts defined but none of them have a contact override for the handler, a single handler event is sent.